consul/plan.sh

```diff
@@ -0,0 +1,40 @@
+pkg_origin=core
+pkg_name=consul
+pkg_version=0.7.0
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=("Apache 2")
+pkg_upstream=https://www.consul.io/
+pkg_source=https://releases.hashicorp.com/${pkg_name}/${pkg_version}/${pkg_name}_${pkg_version}_linux_amd64.zip
+pkg_filename=${pkg_name}-${pkg_version}.zip
+pkg_deps=()
+pkg_build_deps=(core/unzip)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+pkg_expose=(8400 8500 8600)
+pkg_svc_user=root
+
+do_download() {
+  wget -O $HAB_CACHE_SRC_PATH/$pkg_filename $pkg_source
```

Comment:
:actually: you should use `download_file` https://www.habitat.sh/docs/reference/plan-syntax/#utility-functions